### PR TITLE
Upgrade to webpack 5

### PR DESCRIPTION
**Pull request by ivard**
This PR upgrades webpack 4 to webpack 5 in all packages we have, to get rid of the deprecation warnings in `npm audit fix`.

I tested it on several browsers and it works fine now. I had to add some missing polyfills. For example, we need the `core-js/modules/es.promise.finally` polyfill to use finally on Promises in old browsers. This missing polyfill must have caused problems in WebPack 4 too, so this can be considered a bug fix. Another bug was that the polling messages were cached on some older browsers. I explicitly disabled caching there now.

I found some more bugs, which I didn't fix:

- EventSource (sse) is not working in IE11. This is a [bug](https://github.com/EventSource/eventsource/issues/88) in that library and is already present since 2017. Because IE11 will be retired in June 2022, we will not implement a work-around.
- There is a scaling issue in the pairing input field that causes the input field to overflow its parent. This is [a known bug with `flex` in IE10 and IE11](https://github.com/philipwalton/flexbugs#flexbug-2). It just works normally, but it looks a bit weird. The proposed work-arounds do not work. because we precisely use it in a way for which there is no easy workaround (box-sizing with margin).

### CODE REVIEW

File: plugins/irma-client/status-listener.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ module.exports = class StatusListener {
   _pollOnce() {
     return (
       // eslint-disable-next-line compat/compat
-      fetch(this._pollingUrl, this._fetchParams)
```
Comment (sietseringers) [inline plugins/irma-client/status-listener.js:129]: Is this cache addition related to the webpack upgrade?

**Review by sietseringers — APPROVED** [review]

File: plugins/irma-client/status-listener.js (hunk 1 above)
Comment (ivard) [reply plugins/irma-client/status-listener.js:129]: I did not explicitly test it, but I'd assume this currently is a problem on `master` too. I cannot imagine `webpack` resolving such an issue.
